manifest.json

```diff
@@ -1,20 +1,25 @@
 {
+  "manifest_version": 2,
+  "name": "OpenInSteam",
+  "version": "1.3",
 
-    "manifest_version": 2,
-    "name": "OpenInSteam",
-    "version": "1.2",
-  
-    "description": "Adds a button to open the current Steam page inside Steam.",
-  
-    "icons": {
-      "48": "icons/openinsteam-48.png"
-    },  
+  "description": "Adds a button to open the current Steam page inside Steam.",
 
-    "content_scripts": [
-      {
-        "exclude_matches": ["*://store.steampowered.com/cart/*", "*://store.steampowered.com/checkout/*"],
-        "matches": ["*://store.steampowered.com/*", "*://steamcommunity.com/*"],
-        "js": ["openinsteam.js"]
-      }
-    ]
-  }
\ No newline at end of file
+  "icons": {
+    "48": "icons/openinsteam-48.png"
+  },
+  "permissions": ["menus", "tabs"],
```

Comment:
I think `menus` permission is FireFox specific, use `contextMenus` permission so it also works for other browsers. Then you also need to change `browser.menus` to `browser.contextMenus` in the code.

Reply:
I can work on that. I see that the README only lists Firefox as a supported browser for the extension. Is there a reason why it's not published on Chrome as well?

Reply:
The reason it's only on FireFox is that I myself primarily uses FireFox. Also, I don't want to maintain stuff on Chrome if no one is using it. 😄 I've only tried it on FireFox, so I don't even know if it works on other stuff. But I still think it's a good idea to try and be cross platform, if it's an easy change. Maybe someday I'll make a Chrome version.